app/Models/Word.php

```diff
@@ -15,6 +15,7 @@ class Word extends Model
         'text',
         'icon',
         'color',
+        'user_id'
```

Comment:
like above, we might not need this property is we get words through users (and since we have a `belongsTo` relation, we can get a word's owner like `$word->user()->get()` anyway)